apps/language_models/scripts/vicuna.py

```diff
@@ -103,7 +103,14 @@
 )
 
 
-def brevitas〇matmul_rhs_group_quant〡shape(lhs: List[int], rhs: List[int], rhs_scale: List[int], rhs_zero_point: List[int], rhs_bit_width: int, rhs_group_size: int) -> List[int]:
+def brevitas〇matmul_rhs_group_quant〡shape(
```

Comment:
Wouldn't you have the signature mismatch error with this change?

Reply:
Also we should remove the unicode `〇`